src/components/internals/multivalue/multivalue.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { css } from '@emotion/core';
+import PropTypes from 'prop-types';
+import Tag from '../../tag';
+
+const MultiValue = props => (
+  <div
+    css={css`
+      display: inline-block;
+      margin: 2px;
+    `}
+  >
+    <Tag
+      styles={{
+        body: css`
+          padding: 4px 8px 3px 6px;
```

Comment:
If we don't want to do math based on current sizing vars we should maybe add a comment to say why we need this?

Reply:
I was going to come back to the math and or comment writing tomorrow. Sorry.